extensions/shared.tsconfig.json

```diff
@@ -8,6 +8,7 @@
 		"noImplicitReturns": true,
 		"noUnusedLocals": true,
 		"noUnusedParameters": true,
-		"forceConsistentCasingInFileNames": true
+		"forceConsistentCasingInFileNames": true,
+		"sourceMap": true
```

Comment:
don't need this, this is ignored during compilation anyways

Reply:
Which option? sourceMap or forceConsistentCasing?

Reply:
sourcemaps, we use gulp-sourcemaps to generate them and it just ignores this setting. extensions will get sourcemaps without this setting. so there's no need to introduce the merge conflict.